cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -466,6 +467,16 @@ instance HasSeverityAnnotation (TracePeerSelection addr) where
 
       TraceDemoteBigLedgerPeersAsynchronous {} -> Warning
 
+      TraceUseBootstrapPeersChanged {} -> Info
+      TraceBootstrapPeersFlagChangedWhilstInSensitiveState -> Info
+
+      TraceLedgerStateJudgementChanged {} -> Info
+      TraceOnlyBootstrapPeers {}          -> Info
```

Comment:
I think this should be a `Notice`.  For end users, it doesn't matter as they are not reading log files, but for SPOs this should trigger a warning.

Reply:
Maybe you are right 😄